Library/Formula/d-bus.rb

```diff
@@ -12,6 +13,13 @@ class DBus < Formula
     sha256 "08094b506bf81ce24aa9c38d9414b43726b35d19461440ecfc74d47d02298d50" => :mavericks
   end
 
+  # Patch applies the config templating fixed in https://bugs.freedesktop.org/show_bug.cgi?id=94494
+  # Homebrew pr/issue: 50219
+  patch do
+    url "https://raw.githubusercontent.com/homebrew/patches/master/d-bus/org.freedesktop.dbus-session.plist.osx.diff"
```

Comment:
Please use the commit reference rather than `master`. Means if it ever gets accidentally removed this formula doesn't start randomly dying on `fetch` for people.